src/frontends/paddle/src/op/flip.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs flip(const NodeContext& node) {
+    const auto data_node = node.get_input("X");
+    const auto axes = node.get_attribute<std::vector<int32_t>>("axis");
+    const auto input_shape = data_node.get_partial_shape().get_shape();
```

Comment:
If it is a dynamic shape model (i.e. some dimension in the input shape is unknown until inference time), the solution is not feasible, and there would be an exception to call get_shape().
So, could you please -
1. add this static shape check, as the first step.
2. the second step, to investigate how to support dynamic shape input.

Reply:
Use `VariadicSplit` will  be able to support dynamic shape. I will update after passing tests.

Reply:
Hi, I think it hard to support dynamic shape input, cause we need to split tensor into  several ones which have one dimension in specified axis. 
But if there has some op supports automatically spliting tensor into one-dim-tensor, instead of specifing split numbers(for example, the distinction of torch.split and torch.chunk). It will become eazier.

Reply:
As far as I can see, you really need shape dimensions to be static only on `axes` dimensions. I recommend removing `get_shape()` call here and instead call `axes[idx].get_length()` in the loop below where needed. Also please add `PADDLE_OP_CHECK` for `axes[idx].is_static()` for each call of `get_length()`.

To support dynamic shape on axis you could try using `Range` with -1 step to get indices and `Gather` on that indices